web/javascript/torrent-row.js

```diff
@@ -107,6 +107,14 @@ TorrentRendererHelper.formatDL = function (t) {
     return '↓ ' + Transmission.fmt.speedBps(t.getDownloadSpeed());
 };
 
+TorrentRendererHelper.formatETA = function(t)
+{
+        var eta = t.getETA();
+        if (eta < 0 || eta >= (999 * 60 *60))
+                return "na";
```

Comment:
This will look odd. When the number is fine to display, it's "ETA: 5 days", but when it's not it's just "na"... "na" what?.. I'd rather not display anything at all.